AWS_Open_Data/README.md

```diff
@@ -272,3 +273,86 @@ Data are organized by equipment type and date and objects' prefixes are formed a
   - `SN` is the instrument serial number
   - `ext` is the file extension and format `[evt|csd|xml|csv.gz]`
   
+
+
+## <a name="tseries"></a>Time Series data
+
+Time series data derived from the GeoNet sensor network. 
+
+Data are provided in commonly used text open file format and include:
+
+- [Tilde time series data](#tilde)
+
+Additional information around this dataset can be found on the [GeoNet website](https://www.geonet.org.nz/).
+
+All time series data are available under the bucket and prefix `s3://geonet-open-data/time-series/`.
+
+### <a name="tilde"></a>Tilde Time Series data
+Time series data derived from GeoNet sensor network, repackaged and formatted via the GeoNet Tilde application.
+
+Details on Tilde data name format can be found on the [GeoNet Tilde API website page](https://tilde.geonet.org.nz/).
+Details on available dataset can be found on the [GeoNet Tilde Data Discovery website page](https://tilde.geonet.org.nz/ui/data-exploration).
+
+This dataset is the same that can be obtained from the Tilde API _data_ endpoint. For large data requests and bulk downloads, users are recommended to download data from this AWS Open Data bucket.
+
+Data are provided in gzip compressed csv (Comma Separated Values) format.
+
+All time series data are available under the bucket and prefix `s3://geonet-open-data/time-series/tilde/`.
+
+prexix | content
+-- | --
+`time-series/tilde/` | Tilde time series data
+
+Tilde application has only been released in 2021 and we expect that the format of this dataset might change in the future. To allow our users to migrate their downstream procedures, object prefixes will include versioning. 
+
+For this dataset, object prefixes are following an _hive style_ partitioning. 
+
+Data are organized by version, domain, station, time series name, station sensor code, time series parameters (method and aspect) and time. 
```

Comment:
I think it should just say `sensor code`, not `station sensor code`.

Reply:
done here and above